src/AVKit/Enums.cs

```diff
@@ -31,16 +31,32 @@ public enum AVPlayerViewTrimResult : long {
 		OKButton,
 		CancelButton
 	}
+	
+	[Mac (10,15)]
+	[Native]
+	public enum AVRoutePickerViewButtonState : long
+	{
```

Comment:
The brace should be on the same line as the type. https://www.mono-project.com/community/contributing/coding-guidelines. See the little summary table on that page. If you're using Visual Studio for Mac, you can setup the coding guidelines to `mono` and it will do that automatically for you.